src/auth.rs

```diff
@@ -30,7 +30,7 @@ pub fn require_register_provider() -> Result<(), String> {
 }
 
 pub fn is_rpc_allowed(caller: &Principal) -> bool {
-    METADATA.with(|m| m.borrow().get().open_rpc_access) || is_authorized(caller, Auth::Rpc)
+    METADATA.with(|m| m.borrow().get().open_rpc_access) || is_authorized(caller, Auth::PriorityRpc)
```

Comment:
This was supposed to be named `PriorityRpc` instead of `Rpc`, but this change was apparently lost in some unmerged PR a while ago.